roles/kalite/tasks/install.yml

```diff
@@ -15,6 +15,7 @@
   package: 
     name: python2
     state: present
+  when: is_ubuntu_20 | bool
```

Comment:
We might be stuck with KA Lite a lot longer that we all imagined.  So can we please change:

```
when: is_ubuntu_20 | bool
```

...to something like:

```
when: is_ubuntu and not is_ubuntu_18 and not is_ubuntu_19
```